### DISCUSSION TITLE
树状数据层次化方法CascadingTree会丢弃找不到父节点的数据

### DISCUSSION THREAD

**Discussion by pankey888** [dotnetcore/BootstrapBlazor#3140] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

以组织机构为例，若用户只能看到某公司的其中一个下属公司及其子部门，那么该下属公司的父节点Id并不为空，但使用CascadingTree进行层次化处理该数据时，会出现整个下属公司及其子部门都被丢弃。如图所示，在示例数据中Navigation two添加ParentId = "1000"：
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/52560244/d22da26b-2990-4bd4-b15c-d0e3e138dc70)
其显示结果中将丢弃Navigation two：
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/52560244/12a0b328-2801-4ba3-8d9d-51119afd004d)

### Expected Behavior

期望将无法找到父节点的数据放在顶级。

**Comment by bb-auto** [comment]
@pankey888 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@pankey888 你这是业务逻辑，组件库不负责这些，你的需求是放到根上，有些人的需求可能是放到一个 无父节点的一个节点上

**Reply by pankey888** [reply]
但是如果遇到类似这类业务，就没办法呈现数据了，不太可能强制将下属公司的父节点Id设为null。

**Reply by ArgoZhang** [reply]
别用这个方法不就行了吗？自己写个层次化的就可以了啊

**Reply by pankey888** [reply]
已解决了，谢谢
